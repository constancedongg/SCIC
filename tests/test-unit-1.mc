/*
test local var declaration with unit
*/

int '{m} x;
float '{cm} y;
int '{m} func main(){
    x = 2;
    int '{cm} z = 1;
    print(z);
    int '{m} w = x;
    print(x);
    print(w);
    printl("in main");
    return 0;
}