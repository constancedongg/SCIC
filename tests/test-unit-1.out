1
2
2
in main